### PR TITLE
Upload and publish test results

### PR DESCRIPTION
**Pull request by luckslovez**
In this PR:
1. Upload combined test report to pre_release (from the ansible controller)
2. Download the test report (from GH workflow)
3. Leverage actions to publish results in PR (same GH workflow)

### CODE REVIEW

File: integration-tests/test-suite/package.json [hunk 1]
```diff
@@ -3,7 +3,8 @@
   "version": "1.0.0",
   "description": "Validates fluent bit end-to-end with several configurations",
   "scripts": {
-    "test": "jest --verbose fluent-bit-tests.test.js"
+    "test-all": "jest --verbose",
+    "test-common": "jest --verbose common.test.js"
```
Comment (jsubirat) [inline integration-tests/test-suite/package.json:7]: `"test": "jest --verbose"`

File: ansible/provision-and-execute-tests/playbook.yml [hunk 2]
```diff
@@ -215,7 +232,7 @@
         state: absent
 
     - name: Run test-suite
-      ansible.windows.win_shell: 'npm run test'
+      ansible.windows.win_shell: 'npm run test-all'
```
Comment (jsubirat) [inline ansible/provision-and-execute-tests/playbook.yml:235]: `test`

File: ansible/provision-and-execute-tests/playbook.yml [hunk 3]
```diff
@@ -82,7 +90,7 @@
         state: absent
 
     - name: Run test-suite
-      ansible.builtin.shell: "source ~/.nvm/nvm.sh && npm run test"
+      ansible.builtin.shell: "source ~/.nvm/nvm.sh && npm run test-common"
```
Comment (jsubirat) [inline ansible/provision-and-execute-tests/playbook.yml:93]: `test`

**Review by jsubirat — COMMENTED** [review]
Excellent work! Some comments ahead

File: ansible/provision-and-execute-tests/roles/install_nr_fluent_bit_output/tasks/linux.yml [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+  ansible.builtin.service:
+    name: newrelic-infra
+    state: stopped
+  become: true
+
+- name: Download NR FB Output Plugin
+  ansible.builtin.get_url:
+    url: "{{ nr_fb_output_plugin_url }}"
+    dest: "{{ nr_fb_output_plugin_download_file_path }}"
+  become: true
```
[the remark this answers is not in the document]
Comment (luckslovez) [reply ansible/provision-and-execute-tests/roles/install_nr_fluent_bit_output/tasks/linux.yml:14]: Nice catch, ideally we shouldn't let me try

**Review by jsubirat — APPROVED** [review]
🚀 🎉

**Pull request merged**